catroid/src/androidTest/java/org/catrobat/catroid/test/content/actions/GoNStepsBackActionTest.java

```diff
@@ -67,12 +72,15 @@ public void setUp() throws Exception {
 
 		background = new SingleSprite("background");
 		parentGroup.addActor(background.look);
-		sprite = new SingleSprite("testSprite");
-		parentGroup.addActor(sprite.look);
-		project.getDefaultScene().addSprite(sprite);
-		sprite2 = new SingleSprite("testSprite2");
-		parentGroup.addActor(sprite2.look);
-		project.getDefaultScene().addSprite(sprite2);
+		penActorSprite = new SingleSprite("penActor");
+		parentGroup.addActor(penActorSprite.look);
+		project.getDefaultScene().addSprite(penActorSprite);
+		embroideryActorSprite = new SingleSprite("embroideryActor");
+		parentGroup.addActor(embroideryActorSprite.look);
+		project.getDefaultScene().addSprite(embroideryActorSprite);
+		realSprite = new SingleSprite("testSprite");
+		parentGroup.addActor(realSprite.look);
+		project.getDefaultScene().addSprite(realSprite);
```

Comment:
Only my opinion: add empty lines to group the initialization of the different sprites to make it easier to read.

Reply:
done